### PR TITLE
Add directories and files for Ansible on AWS

### PR DESCRIPTION
**Pull request by ariordan-redhat**
Create directories, files, and symbolic links for Ansible on AWS docs
so that
we have a framework in the repo to add content to when the Ansible on AWS docs initiative starts.

Use the checklist in the [_Creating a new guide_](https://github.com/RedHatInsights/red-hat-ansible-automation-platform-documentation/wiki/Creating-a-new-guide) wiki article.

Title affected: `/titles/aap-on-aws/`

### CODE REVIEW

File: downstream/titles/aap-on-aws/master.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+include::attributes/attributes.adoc[]
+
+
+// Book Title
+= {AAPonAWSName} guide
+
+Thank you for your interest in {PlatformName}. {PlatformNameShort} is a commercial offering that helps teams manage complex multi-tier deployments by adding control, knowledge, and delegation to Ansible-powered environments.
+
+This guide helps you to understand the installation and use of {AAPonAzureNameShort} apps on cloud platforms. This document has been updated to include information for the latest release of {AAPonAzureNameShort}.
```
Comment (lmalivert) [inline downstream/titles/aap-on-aws/master.adoc:15]:
```suggestion
This guide helps you to understand the installation and use of {AAPonAWSNameShort} apps on cloud platforms. This document has been updated to include information for the latest release of {AAPonAWSNameShort}.
```

File: downstream/modules/aap-on-aws/con-aap-aws-about.adoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
+////
+Base the file name and the ID on the module title. For example:
+* file name: con-my-concept-module-a.adoc
+* ID: [id="con-my-concept-module-a_{context}"]
+* Title: = My concept module A
+////
+
+[id="con-aap-aws-about"]
+
+= About {AAPonAzureName}
```
Comment (lmalivert) [inline downstream/modules/aap-on-aws/con-aap-aws-about.adoc:10]:
```suggestion
= About {AAPonAWSName}
```

**Review by lmalivert — APPROVED** [review]

**Comment by lmalivert** [comment]
@ariordan-redhat Looks good! I few comments where I think there should be `AWS` instead of `Azure`

**Comment by ariordan-redhat** [comment]
Thanks for the review, @lmalivert.

**Pull request merged**